content/sensu-go/5.16/guides/monitor-sensu.md

```diff
@@ -0,0 +1,150 @@
+Distributed single backend instances (federated) | Service checks, port checks | Plugin name |
+Distributed clustered backend instances (federated) | Prometheus collector check, API queries as checks | Plugin name |
+
+### Monitor the Sensu backend locally
+
+Monitor the host that the `sensu-backend` process runs on like any other node in your infrastructure.
+This includes checks and metrics for [CPU][1], [memory][2], [disk][3], and [networking][4].
+Find more plugins at [Bonsai, the Sensu asset index][5].
+
+### Monitor the Sensu backend remotely
+
+Monitor the `sensu-backend` server process from an independent Sensu instance.
+To do this, call Sensu's [health API endpoint][6] and use the [check-http plugin][7] with the following check definition:
+
+{{< highlight json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "namespace": "default",
+    "name": "check_sensu_server_port"
+  },
+  "spec": {
+    "command": "check-http.rb -h remote-api-hostname -P 8080 -p /health --response-code 204",
+    "subscriptions": [
+      "monitor_remote_sensu_api"
+    ],
+    "interval": 60,
+    "publish": true
+  }
+}
+{{< /highlight >}}
+
+## Monitor your Sensu agent instances
+
+You can use API port checks to monitor your Sensu agent.
+Service checks are not effective for monitoring the Sensu agent because if the agent is down, a service check for it won't run.
+
+The [keepalive][11] is a built-in monitor for whether the Sensu agent service is running and functional, although network issues can cause a non-OK keepalive event even if the agent is functional.
+
+## Monitor your Sensu API instances
+
+To monitor your Sensu API instances from an independent Sensu instance, call the port that the API is listening on using the [check-port plugin][8] with the following check definition:
+
+{{< highlight json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "namespace": "default",
+    "name": "check_sensu_api_port"
+  },
+  "spec": {
+    "command": "check-ports.rb -H remote-sensu-server-hostname -p 8080",
+    "subscriptions": [
+      "monitor_remote_sensu_api"
+    ],
+    "interval": 60,
+    "publish": true
+  }
+}
+{{< /highlight >}}
+
+## Monitor your Sensu dashboard instances
+
+To monitor your dashboard instances with a remote network port check, use the [check-port plugin][8] with the following check definition:
+
+{{< highlight json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "namespace": "default",
+    "name": "check_dashboard_port"
+  },
+  "spec": {
+    "command": "check-ports.rb -H dashboard-remote-hostname -p 3000",
+    "subscriptions": [
+      "monitor_remote_dashboard_port"
+    ],
+    "interval": 60,
+    "publish": true
+  }
+}
+{{< /highlight >}}
+
+## External etcd PLACEHOLDER
+
+Note from Richie: We may want to eventually set up a section for the other possible setups (external etcd, postgres).
```

Comment:
We should check with Engineering if there is any value in a backend monitoring its own etcd availability or if the health endpoint is enough from another cluster.

Reply:
Action item in document.